manifests/0000_20_kube-apiserver-operator_00_cr-anyuid.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  creationTimestamp: null
```

Comment:
nit: remove creationTimestamp in the yamls

Reply:
:man_facepalming: fixed.